fastestimator/pipeline.py

```diff
@@ -330,7 +379,11 @@ def get_results(self, mode: str = "train", epoch: int = 1, num_steps: int = 1,
                 results = results[0]
             return results
 
-    def __call__(self, mode: str, epoch: int = 1, shuffle: Optional[bool] = None,
+    def __call__(self,
+                 mode: str,
+                 epoch: int = 1,
+                 ds_id: Union[None, str] = None,
```

Comment:
```suggestion
                 ds_id: Optional[str] = None,
```

Reply:
I guess default None is good here so that users without multi dataset don't have to think about it